weave-js/src/components/PagePanelComponents/Home/HomePreviewSidebar.tsx

```diff
@@ -148,21 +192,111 @@ export const HomePreviewSidebarTemplate: React.FC<{
   );
 };
 
+const Loader = () => (
```

Comment:
This is pulled from lower in the file, just made it into a component so i could use it multiple spots